toolchain/install/symlink_helpers.bzl

```diff
@@ -75,3 +105,35 @@ symlink_file = rule(
         "symlink_label": attr.label(allow_single_file = True),
     },
 )
+
```

Comment:
Did you make any changes below here? I ask because I don't see any changes compared to the code that was previously at the top of the file, but if there aren't any, it's bizarre that the diff is treating it as new code.